test/k8s/res_limits_test.go

```diff
@@ -0,0 +1,512 @@
+package k8s_test
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"github.com/ghodss/yaml"
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"k8s.io/apimachinery/pkg/watch"
+	"os"
```

Comment:
format

Reply:
`make check` will validate the format. So I assume it's good.